app/scripts/controllers/SysinfoController.coffee

```diff
@@ -103,5 +105,16 @@ angular.module('neo4jApp.controllers')
       $scope.autoRefresh = !$scope.autoRefresh
       refreshLater()
 
+    $scope.isCurrentConnection = (addresses) ->
+      if Settings.useBolt
+        boltHost = Settings.boltHost || $rootScope.boltHost
+        boltAddressForMember = Utils.getServerAddressByProtocol('bolt', addresses)[0]
+        return boltAddressForMember && boltAddressForMember.includes(boltHost)
```

Comment:
Hmm - this is a really odd case where host has an implicit port of 7687. Not sure how we should handle this - should we look for '7687' as a known default?